packages/suite/src/constants/suite/routes.ts

```diff
@@ -1,223 +1,8 @@
 import { ArrayElement, ConstWithOptionalFields } from '@suite/types/utils';
```

Comment:
This file only has types in it, so maybe route/types?

Reply:
Types are based of route config and only used there. I think it's fine to leave this as is.